### DISCUSSION TITLE
Default Game Type for Player Pages

### DISCUSSION THREAD

**Discussion by Elitesparkle** [Heroes-Profile/heroesprofile#918] [Ideas]
The selection for Default Game Type in Site Settings isn't being used for all the Player Pages: Player Profile, Friend and Foe, Hero Stats, Role Stats, Map Stats, Matchups, Talents, Match History.

**Comment by Zemill** [comment]
While using the site it was consistently confusing when I had my default game mode set to Storm League, going to players profiles and not seeing all of their data.  So I made the decision to change Player data to all game modes and all seasons.

I could break out the settings options to specify a difference between defaults for global pages and player pages

**Comment by Zemill** [comment]
This has been added
